### DISCUSSION TITLE
Please enlarge the edit fields

### DISCUSSION THREAD

**Discussion by UweSchwo** [elgatosf/elgato#13] [Ideas]
When editing the Stream Deck keys I see very small edit fields, they are short and usually have only one visible line.
This concerns the for instance title field and even worse the command line field for the "Open" button. 

Please give the fields more space for a better overview!

Btw. hotkeys or at least Tabs for navigation between the fields would be very helpful.

**Comment by ChekTek** [comment]
Hi!

Thank you for the feedback, I will discuss this with the team. We do give third-party developers the option to use an input element as well as a textfield element, which would provide more space for user input. 

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea

The tab index is also something provided by the developer, but I will make a note that we should add this to our plugins internally as well as to the style guide for third party developers. 

Best,
Zack

**Reply by UweSchwo** [reply]
Thanks! 

Concerning key navigation: The current  TAB-Key handling inside the title field (insert a TAB space inside title text) seems to be useless. 
Having at least the TAB key as a navigation key between the fields (including the font and text color setting) would be great. 

Much better of course were some Hotkeys, i.e. ALT+C - title Text color, ALT-P - background picture a. s. o.
At the moment I use AHK with picture recognition (ImageSearch) to have some automation concerning the SD keys appearance (for instance enlarge title font size by only one button press per SD key). 

Even a "backup all profiles" key seems to be useful.
